packages/flutter/lib/src/material/app.dart

```diff
@@ -770,6 +770,11 @@ class MaterialApp extends StatefulWidget {
   }
 }
 
+/// Scroll behaviour that scrolls by all pointer devices.
+final ScrollBehavior kAllDraggableScrollBehavior =
```

Comment:
I don't know that we would want to define this as a constant in the framework, especially since it is specific to only MaterialScrollBehavior.